### PR TITLE
Adding Predictive Analytics RMD file

### PR DESCRIPTION
**Pull request by rjrahul24**
Attached is the .Rmd file for my community contribution.

There is one external package that needs direct installation from GitHub. Details are mentioned in comments. 
require(devtools) 
install_github("jbryer/makeR")

### CODE REVIEW

**Comment by rjrahul24** [comment]
@pyropig @Abhimanyu210100 Are any changes required? Unsure why the PR was assigned back to me?

**Comment by pyropig** [comment]
> @pyropig @Abhimanyu210100 Are any changes required? Unsure why the PR was assigned back to me?

I'm not the assigner but I guess it's just a miss-click

**Comment by pyropig** [comment]
@Abhimanyu210100 could you please assign to another merger? its seems the branch owner is also in the list of assignees so we might need to take care of that

**Comment by Abhimanyu210100** [comment]
Hi @rjrahul24 apologies for the assignment, it was by mistake

File: predictive-analytics-r.Rmd [hunk 1]
```diff
@@ -0,0 +1,348 @@
+# Predictive Analytics using Data Visualization in R
+
+Rahulraj Singh
+
+```{r setup, include=FALSE}
```
Comment (kq2153) [inline predictive-analytics-r.Rmd:5]: Please change this to "{r, include=FALSE}"

**Review by kq2153 — CHANGES_REQUESTED** [review]
Hi, please make the change requested in the comments. Thanks!

**Review by rjrahul24 — COMMENTED** [review]
@kq2153 Changes reflect in the latest commit. Thanks!

**Comment by rjrahul24** [comment]
Hey @kq2153, just checking back to see if there are any more changes needed from my end?

**Review by kq2153 — APPROVED** [review]

File: predictive-analytics-r.Rmd [hunk 2]
```diff
@@ -0,0 +1,348 @@
+# Predictive Analytics using Data Visualization in R
+
+Rahulraj Singh
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r Package Imports, echo=FALSE, include=TRUE}
```
[the remark this answers is not in the document]
Comment (alexnsaunders) [reply predictive-analytics-r.Rmd:9]: > @jtr13 just wanted to double check, but will having "include = TRUE" on these packages cause any issues in the bookdown?

Yes this is causing an issue looks like it needs to be installed directly in github which is beyond the limits of my knowledge - we can do or we will have to take out - package makeR not on CRAN anymore @jtr13

**Comment by jtr13** [comment]
I've only seen `include = FALSE` in use but `include=TRUE` won't affect anything -- it just controls whether the code and output are displayed in the rendered version. 

If a package is not available on CRAN but is on GitHub, the remote needs to be specified at the end of the `DESCRIPTION` file. Note that the package name appears in two places: both in the Imports section and in the remotes section.

**Comment by kq2153** [comment]
I added the github remote reference to the end of the Description file, uncommented this Rmd file and it seems the bookdown compiled!

**Comment by jtr13** [comment]
Woohoo, nice work! 🎉  You can close any issues related to the problem that you fixed.

**Pull request merged**